### ISSUE TITLE
Addons: /market/addons fails when an instance is not connected

### ISSUE THREAD

**Issue by Innders** [ynput/ayon-backend#232]
## Story

My instance is not connected to Ynput cloud yet and I go to the addon market.

### Expected behavior

I can see all addons but I can't install them.

### Error message

`[GET] /api/market/addons`

```json
{
  "code": 500,
  "detail": "Internal server error",
  "traceback": "Client error '404 Not Found' for url 'https://im.ynput.cloud/api/v1/market/addons'\nFor more information check: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404",
  "path": "[GET] /market/addons",
  "file": "/usr/local/lib/python3.11/site-packages/httpx/_models.py",
  "function": "raise_for_status",
  "line": 759
}
```

**Comment by martastain** [comment]
Is this fixed by https://github.com/ynput/ayon-backend/commit/16172eb3fc71e0362599731292844637d918ef1e ?

**Comment by Innders** [comment]
Fixed by [16172eb](https://github.com/ynput/ayon-backend/commit/16172eb3fc71e0362599731292844637d918ef1e)
